docs/core/resilience/http-resilience.md

```diff
@@ -0,0 +1,228 @@
+// If you're using a generic host, you'll instead use its service collection:
+//
+// var host = Host.CreateApplicationBuilder(args);
+// host.Services.AddHttpClient<ExampleClient>(client => { ... });
+
+var services = new ServiceCollection();
+
+var builder = services.AddHttpClient<ExampleClient>(
+    configureClient: static client =>
+    {
+        client.BaseAddress = new("https://jsonplaceholder.typicode.com");
+    })
+    .AddStandardResilienceHandler();
+```
+
+The preceding code:
+
+- Adds an <xref:System.Net.Http.HttpClient> for the `ExampleClient` type to the service container.
+- Configures the <xref:System.Net.Http.HttpClient> to use `"https://jsonplaceholder.typicode.com"` as the base address.
+- Adds the standard resilience handler to the <xref:System.Net.Http.HttpClient>.
+- Declares a `builder` (of type `IHttpStandardResilienceHandlerBuilder`), which is used to configure the standard resilience handler. There are extension methods to configure the standard resilience handler.
+
+### Standard resilience handler defaults
+
+The default configuration chains five resilience strategies in the following order (from the outermost to the innermost):
+
+| Order | Strategy | Description |
+|--:|--|--|
+| **1** | Rate limiter | The rate limiter pipeline limits the maximum number of concurrent requests being sent to the dependency. |
+| **2** | Total request timeout | The total request timeout pipeline applies an overall timeout to the execution, ensuring that the request, including retry attempts, doesn't exceed the configured limit. |
+| **3** | Retry | The retry pipeline retries the request in case the dependency is slow or returns a transient error. |
+| **4** | Circuit breaker | The circuit breaker blocks the execution if too many direct failures or timeouts are detected. |
+| **5** | Attempt timeout | The attempt timeout pipeline limits each request attempt duration and throws if it's exceeded. |
+
+## Add standard hedging handler
+
+The standard hedging handler wraps the execution of the request with a standard hedging mechanism. Hedging retries slow requests in parallel. The standard hedging handler is added by calling the `AddStandardHedgingHandler` extension method.
+
+To use the standard hedging handler, call `AddStandardHedgingHandler` extension method. The following example configures the `ExampleClient` to use the standard hedging handler.
+
+```csharp
+// A service collection is manually instantiated for this example.
+// If you're using a generic host, you'll instead use its service collection:
+//
+// var host = Host.CreateApplicationBuilder(args);
+// host.Services.AddHttpClient<ExampleClient>(client => { ... });
+
+var services = new ServiceCollection();
+
+var builder = services.AddHttpClient<ExampleClient>(
+    configureClient: static client =>
+    {
+        client.BaseAddress = new("https://jsonplaceholder.typicode.com");
+    })
+    .AddStandardHedgingHandler();
```

Comment:
How about routing aspects of hedging? The ability to configure different routes for individual requests?

Reply:
Done